test/test-tcp-exportimport.c

```diff
@@ -0,0 +1,240 @@
+/* Copyright Joyent, Inc. and other Node contributors. All rights reserved.
```

Comment:
Can you use this: https://github.com/libuv/libuv/blob/a629688008694ed8022269e66826d4d6ec688b83/test/test-process-title-threadsafe.c#L1

Reply:
This file was brought back from limbo - it used to be named test-ipc-threads.c, it was removed with uv_export/uv_import in 2012, I retained the original copyrights note on it as I just changed the name of the test and updated the APIs to libuv v1.x